network_windows.go

```diff
@@ -30,6 +30,10 @@ func (n *network) startResolver() {
 		options := n.Info().DriverOptions()
 		hnsid := options[windows.HNSID]
 
+		if hnsid == "" {
+			return
+		}
+
```

Comment:
@msabansal Is this an error case ? Looks like its not. May be we can avoid calling startResolver() if we know up front its not needed in certain cases (specific to the network ?)